content/docs/1.0.0/tutorial/deploy-your-microservice.md

```diff
@@ -10,8 +10,8 @@ While you can use a custom registry (or the public [Docker Hub](https://hub.dock
 
 To get a list of the container registries, we can use `registry list`:
 
-```sh
-iofog-controller registry list
+```bash
+docker exec -it iofog-controller iofog-controller registry list
 ```
 
 You should see two, the first being [Docker Hub](https://hub.docker.com/), but we're going to use the second one, which is the built-in private registry:
```

Comment:
**We** should see two **registries.** **The first is Docker Hub and the second is the built-in private registry, which we're going to use.**